src/components/BlogPost.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import Post from './Post';
+import ListOfComments from './ListOfComments';
+
+interface BlogPostProps {
+    title: string;
+    author: string;
+    content: string;
+    claps: number;
+    id: number;
+}
+
+const BlogPost: React.FC<BlogPostProps> = ({title, author, content, claps, id}) => {
+    return(
+        <Container>
+            <Post title={title} author={author} content={content} claps={claps}/>
+            <ListOfComments />
+        </Container>
+    );
+}
```

Comment:
```suggestion
const BlogPost: React.FC<BlogPostProps> = ({title, author, content, claps, id}) => (
        <Container>
            <Post title={title} author={author} content={content} claps={claps}/>
            <ListOfComments />
        </Container>
    );
```

Reply:
Don't use github to commit this ^

Reply:
what you mean?

Reply:
nvm i understand

Reply:
I think keeping the return element helps to show what react is essentially doing under the hood

Reply:
Hmm, that's fair, but if we want to teach best practices this is the way to do it. Some linters even enforce that actually.

Reply:
I think we should leave out the return, but when covering lambdas in the script, be sure to include [both styles](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions#The_arrow_function_expression_%3E)

Reply:
sounds good